nix/cardano-services/deployments.nix

```diff
@@ -0,0 +1,57 @@
+let
+  inherit (inputs.std) dmerge;
+  inherit (inputs.std.inputs) haumea;
+  inherit (inputs.std.lib.ops) readYAML;
+
+  loadYaml = _: _: readYAML;
+
+  baseline = with haumea.lib;
+    load {
+      src = ./deployments;
+      loader = [(matchers.regex ''^.+\.(yaml|yml)'' loadYaml)];
+    };
+in {
+  dev-preview = dmerge baseline {
+    meta.description = "Development Environment on the Cardano Preview Chain (Frankfurt)";
+    backend-deployment = {
+      spec.template.spec.containers = dmerge.updateOn "name" [
+        {
+          name = "backend";
+          image = cell.oci-images.cardano-services.image.name;
+        }
+      ];
+    };
+    coingecko-proxy-deployment = {
+      spec.template.spec.containers = dmerge.updateOn "name" [
+        {
+          name = "coingecko-proxy";
+          image = cell.oci-images.cardano-services.image.name;
+        }
+      ];
+    };
```

Comment:
I don't think that this should be here

Reply:
Indeed not. :smile: And the coingecko proxy should move into this repo consequentially, too. At least judging by the architectural criterion. I backlogged with due urgency [LW-8023](https://input-output.atlassian.net/browse/LW-8023) for follow up.

Reply:
Addressed (removed only) in [force-push](https://github.com/input-output-hk/cardano-js-sdk/compare/10167f20f9c7d8e8870f84755ab2e0bd1a902e2f..47ce6e3ec1f74f224a253a6c05706323d0dd3ba9)